pkg/updatestrategy/node_pool_manager.go

```diff
@@ -208,7 +208,7 @@ func (m *KubernetesNodePoolManager) taintNode(node *Node, taintKey, taintValue s
 		return nil
 	}
 
-	backoffCfg := backoff.WithMaxTries(backoff.NewConstantBackOff(1*time.Second), maxConflictRetries)
+	backoffCfg := backoff.WithMaxRetries(backoff.NewConstantBackOff(1*time.Second), maxConflictRetries)
```

Comment:
Are these changes triggered by version updates of dependencies?

Reply:
Yes, sorry, I'll do a separate PR with dependency updates to not have everything included here.